### PR TITLE
Installer scripts for DCFABRIC-SUITE PACKS

### PR DESCRIPTION
**Pull request by uday83**
New files to address dcfabric-suite installation
dcfabric-suite-installer-deb.sh
dcfabric-suite-installer-e17.sh
dcfabric-suite-installer-e16.sh

Modified the installer files from ipfabric to dcfabric
bwc-installer.sh and bwc-suite-installer.sh

### CODE REVIEW

**Comment by uday83** [comment]
Test Logs for the install scripts for dcfabric-suites
------------------------------------------------------
[test_install_script_with_suite_name.txt](https://github.com/StackStorm/bwc-installer/files/651739/test_install_script_with_suite_name.txt)
[test_install_script_without_suite_name.txt](https://github.com/StackStorm/bwc-installer/files/651740/test_install_script_without_suite_name.txt)
[test_install_suite_script_with_suite_name.txt](https://github.com/StackStorm/bwc-installer/files/651741/test_install_suite_script_with_suite_name.txt)

File: scripts/bwc-installer.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@ BRANCH='master'
 LICENSE_KEY=''
 SUITE=''
 
-SUITES_LIST=(bwc-ipfabric-suite) # Space separated list of names that should map to package names.
+SUITES_LIST=(dcfabric-suite) # Space separated list of names that should map to package names.
```
Comment (lakshmi-kannan) [inline scripts/bwc-installer.sh]: The prefix `bwc-` was intentionally kept that way. Could you add the bwc- prefix and adjust the file names accordingly?

**Comment by lakshmi-kannan** [comment]
Did you test this on at least one RHEL box and did it work?

**Comment by uday83** [comment]
https://github.com/StackStorm/sandbox-actions/pull/114 is the fix for RHEL/CENTOS issue.

File: scripts/bwc-installer.sh (hunk 1 above)
Comment (samirsss) [inline scripts/bwc-installer.sh]: Spoke to @ramgolla about this and he wants to now have this prefix anymore.

**Pull request merged**